packages/visual-editor/package.json

```diff
@@ -36,14 +34,24 @@
       ],
       "files": [
         "src/**/*.ts",
+        "src/a2/**/*.bgl.json",
         "tests/**/*.ts",
         "tsconfig.json",
         "vite.config.ts",
         "../../core/tsconfig/base.json",
-        "package.json",
-        "langs"
+        "package.json"
       ]
     },
+    "check:tsc": {
+      "command": "tsc -b --pretty --noEmit",
+      "env": {
+        "FORCE_COLOR": "1"
+      },
+      "dependencies": [
+        "typescript-files-and-deps"
+      ],
+      "files": []
```

Comment:
add "output":[] so that wireit will be able to cache this script (by knowing that it doesn't produce any output it needs to track)